File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int countOnes = 0;
        int maxOnes = 0;
        for(int i = 0; i<nums.size(); i++){
            if(nums[i] == 1){
                countOnes++;
                maxOnes = max(countOnes, maxOnes);
            }else{
                countOnes = 0;
            }
        }

        return maxOnes;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxO = 0;
        int cnt = 0;

        for(int i = 0; i<nums.length; i++){
            if(nums[i] == 1){
                cnt++;
            }else{
                cnt=0;
            }

            maxO = Math.max(cnt, maxO);
        }

        return maxO;
    }
}